### DISCUSSION TITLE
Listener Stats Graph

### DISCUSSION THREAD

**Discussion by shaunzyb** [AzuraCast/AzuraCast#3745] [Q&A]
Hi 

Since upgrading to v0.12 Stable from v.0.10.4 I wanted to check the average listener screen is showing as expected.

I did the update on the 29th but it is showing a sharp drop since the 25th.

![Listen Average](https://user-images.githubusercontent.com/78295894/106380531-e6a38b80-63aa-11eb-9b7d-e7b48e27c3bb.jpg)

Does this seem right?

**Comment by inigosastre** [comment]
I've upgraded from 0.15.1 stable to 0.17.1 and I have had the same problem. Is there any solution? Thanks!

<img width="1174" alt="Captura de Pantalla 2022-07-05 a las 13 18 45" src="https://user-images.githubusercontent.com/63013622/177316173-2db81a4a-5d02-43e3-9402-67e6e3f18521.png">
